wrappers/python/openmm/app/data/implicit/gbn.xml

```diff
@@ -0,0 +1,46 @@
+<ForceField>
+ <Script>
+import openmm
+import openmm.app as app
+import openmm.unit as unit
+
+# Find the NonbondedForce.  We need it to look up charges, and also to set the reaction field dielectric to 1.
+
+nonbonded = [f for f in sys.getForces() if isinstance(f, openmm.NonbondedForce)]
+if len(nonbonded) != 1:
+    raise ValueError('Implicit solvent requires the System to contain a single NonbondedForce')
+nonbonded = nonbonded[0]
+nonbonded.setReactionFieldDielectric(1)
+
+# Construct the CustomGBForce.
+
+from openmm.app.internal.customgbforces import GBSAGBnForce
+argMap = {'soluteDielectric':'soluteDielectric', 'solventDielectric':'solventDielectric', 'implicitSolventKappa':'kappa'}
+solventArgs = {'SA':'ACE'}
+for key in argMap:
+    if key in args:
+        solventArgs[argMap[key]] = args[key]
+if nonbondedMethod != app.NoCutoff:
+    solventArgs['cutoff'] = nonbondedCutoff.value_in_unit(unit.nanometers)
+force = GBSAGBnForce(**solventArgs)
+params = GBSAGBnForce.getStandardParameters(topology)
+for i, p in enumerate(params):
+    charge, sigma, epsilon = nonbonded.getParticleParameters(i)
+    force.addParticle([charge, p[0], p[1]])
+force.finalize()
+
+# Set the nonbonded method and cutoff distance.
+
+if nonbondedMethod == app.NoCutoff:
+    force.setNonbondedMethod(openmm.CustomNonbondedForce.NoCutoff)
+elif nonbondedMethod == app.CutoffNonPeriodic:
+    force.setNonbondedMethod(openmm.CustomNonbondedForce.CutoffNonPeriodic)
```

Comment:
I'm not sure anyone really understands how to run GBSA with cutoffs, so it may be sensible to issue a warning here.

Reply:
Here's what the Amber manual says about it:

> If the nonbonded cutoff is used in GB calculations, it should be greater than that for PME calculations, perhaps cut=16.

I'll add a note recommending a larger cutoff.  The theory of applying a cutoff to GB is straightforward, since it's closely related to reaction field.  (Reaction field assumes everything beyond the cutoff distance is a uniform dielectric.  GB assumes everything beyond the Born radius is a uniform dielectric.)

Reply:
I wasn't aware of these recommendations! Sounds like a great opportunity to emit a warning if the cutoff is too small.

Reply:
I was always taught to run GB with no cutoffs.  The cut=16 recommendation seems to be a relic of the days when computers were painfully slow.